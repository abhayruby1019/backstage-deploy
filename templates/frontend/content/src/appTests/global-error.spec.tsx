import { render } from '@testing-library/react'
import GlobalError from '@/app/global-error'

describe('Error Boundary', () => {
  it('should display technical error when type is not passed', () => {
    const { container } = render(<GlobalError />)
    expect(container).toMatchSnapshot()
  })
})
