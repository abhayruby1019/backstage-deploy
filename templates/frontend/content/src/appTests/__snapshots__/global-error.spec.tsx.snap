// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Boundary should display technical error when type is not passed 1`] = `
<div>
  <div
    class="mainError"
  >
    <div
      class="title"
    >
      Technical Error
    </div>
    <a
      class="link"
      href="/"
    >
      Return Home
    </a>
  </div>
</div>
`;
